dbt/models/default/default.vw_pin_sale.sql

```diff
@@ -218,15 +219,15 @@ mydec_sales AS (
             COUNT() OVER (
                 PARTITION BY line_1_primary_pin, line_4_instrument_date
             ) AS num_single_day_sales,
-            year_of_sale
+            year_of_sale AS year
         FROM {{ source('sale', 'mydec') }}
-        WHERE line_2_total_parcels = 1 -- Remove multisales
```

Comment:
I think this line is going to prevent you from having any sales in your mydec sample that are multisales. I don't think we want to exclude multisales unless we invoke the `is_multisale` filter.

Reply:
Ah yep good catch